### PR TITLE
fix(agent): add countDocuments to non-oapi mongodb instrumentation

### PR DESCRIPTION
**Pull request by bduranleau-nr**

### CODE REVIEW

**Comment by zsistla** [comment]
@bduranleau-nr looks great, thanks!
I think both it and the multiverse PR say `This branch is out-of-date with the base branch`.
Can you please update that before any approvals? Otherwise the repo settings might dismiss any approvals after updating the branch.

**Review by zsistla — APPROVED** [review]

**Review by lavarou — APPROVED** [review]

**Pull request merged**
